prov/efa/src/rdm/efa_rdm_cq.c

```diff
@@ -85,18 +86,29 @@ static ssize_t efa_rdm_cq_readfrom(struct fid_cq *cq_fid, void *buf, size_t coun
 {
 	struct efa_rdm_cq *cq;
 	ssize_t ret;
+	struct util_srx_ctx *srx_ctx;
 
 	cq = container_of(cq_fid, struct efa_rdm_cq, util_cq.cq_fid.fid);
 
+	srx_ctx = cq->util_cq.domain->srx->ep_fid.fid.context;
+
+	/* This is a no-op unless domain_attr.threading is FI_THREAD_SAFE */
```

Comment:
No hard opinion but this comment is superfluous. Consider removing.

Reply:
Removed